apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -94,7 +90,8 @@
         return;
```

Comment:
comment for line 93: It's better to use 'warning' or 'error' functions to log such things.